client/webserver/site/src/html/forms.tmpl

```diff
@@ -272,12 +275,12 @@
   The remaining amount may change before the cancel order is matched.
 </div>
 <hr class="dashed mt-2">
-<div class="d-flex flex-row align-items-end pb-4 px-3">
-  <div class="col-12 p-0">
+<div class="d-flex flex-row {{if $passwordIsCached}}justify-content-center{{else}}align-items-end{{end}} pb-4 px-3">
+  <div class="col-12 p-0 {{if $passwordIsCached}}d-hide{{end}}">
```

Comment:
Please don't center align the submit button. Keep it on the right side. This would be consistent with other forms as well.

Use the `visibility` css attribute via bootstrap's `invisible` class so you don't need to set the justification on the parent element.

`<div class="col-12 p-0 {{if $passwordIsCached}}invisible{{end}}">`

Reply:
![Screen Shot 2021-07-15 at 9 38 55 AM](https://user-images.githubusercontent.com/6186350/125797685-3add342d-c6e6-402a-8465-c04ab5b36830.png)
![Screen Shot 2021-07-15 at 9 39 33 AM](https://user-images.githubusercontent.com/6186350/125797693-6e988d46-752a-426f-8341-c9eaf355f7e9.png)

`invisible` gives the large gap you see in the second picture. With `d-hide` and `justify-content-end` it looks like the first one.